components/charts/AnnualizedHousingChart.tsx

```diff
@@ -36,33 +38,7 @@ interface AnnualizedHousingChartProps {
   targetValue?: number;
 }
 
-// Type for bar dataset
-interface BarDataset {
-  type?: "bar";
-  label: string;
-  data: number[];
-  backgroundColor: string;
-  borderColor?: string;
-  borderWidth?: number;
-  hoverBackgroundColor?: string;
-}
-
-// Type for line dataset
-interface LineDataset {
```

Comment:
This and all the duplicates from the other files have been consolidated below, where it uses the interface from Chart.js https://github.com/BuildCanada/OutcomeTracker/blob/a998720f7f9a30fbee9d2d5d318baf4afe84385c/components/charts/types.ts#L9